Sources/SwiftCliCore/Chartify.swift

```diff
@@ -7,16 +7,16 @@ public final class Chartify {
     var chartConstructor: ChartConstructor
     var fileValidator: FileValidator
 
-
     public init(inputValidator: InputValidator, chartConstructor: ChartConstructor, fileValidator: FileValidator) {
 
         self.inputValidator = inputValidator
         self.chartConstructor = chartConstructor
         self.fileValidator = fileValidator
     }
 
-
     public func run(userInput: [String], file: String? = nil, knitFlat: Bool = false) {
+
+
```

Comment:
Do you need these extra lines?  Please review the rest of the PR diff here for extra lines you don't need.